diagnostic.md

```diff
@@ -7,40 +7,41 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git clone https://github.com/ga-wdi-boston/git-basics-diagnostic.git
```

Comment:
`git clone git@github.com/andreasantarl/git-basics-diagnostic.git`. It asked for your forked version's clone path. Also it needs to be the SSH